site/en/external/extension.md

```diff
@@ -113,6 +113,25 @@ def _maven_impl(ctx):
   _generate_hub_repo(name = "maven", repo_attrs)
 ```
 
+### Extension identity
+
+Module extensions are identified by the name and the `.bzl` file that they are
+imported from. In the following example, the extension `maven` is identified by
```

Comment:
maybe say "use" instead of "import", since the directive is `use_extension`?

Reply:
I made it explicit with
> ... that appears in the call to `use_extension`

"that they are used from" seemed ambiguous, as it might be misread as "the file where `use_extension` is called".